participant/app/components/List.tsx

```diff
@@ -0,0 +1,27 @@
+import type { ReactElement } from "react";
+import type { i18nKey } from "app/types";
+import { Trans } from "react-i18next";
+
+export type ListProps = {
+  listKeys: i18nKey[];
+  ordered?: boolean;
```

Comment:
i like to offer a className prop to let people add additional classes to the default ones